### PR TITLE
fix(trading): order single view side

### LINKED ISSUE #3394: Single order view
See also: https://github.com/vegaprotocol/frontend-monorepo/issues/4098

## Story

As a user
I want a way to view all information relating to an order
So that I can inspect the order and trades it created

## Draft Acceptance Criteria

To be refined based on UX Exploration

- [x] I can click an order in the order datagrid and be taken to some UI which shows further details about the order
- [x] I can see all the order data
- [ ] ~~I can see all fills/trades created from the order~~ ([Barney and Matt agreed to omit this]
- [x] I can see if the order was amended and when
- [ ] ~~I can link to the corresponding transaction in explorer~~ (can't do this without the transaction id that we don't have at 
- [ ] ~~I can link to the corresponding transaction in explorer for each order amendment~~ 
- [ ] ~~I can link to the corresponding transaction in explorer for an order cancellation~~ 
(checked with Edd and unfortunately we can't do that without having the transaction IDs attached to the order object)
- [x] I can see if and when the order was cancelled 
- [x] There is a way to get back to the order datagrid
- [x] I can see iceberg order data

## Tasks

- [x] UX Exploration (Incl. sketching and creating / refining tickets to represent the work)
- [x] Team and stakeholder review and refine
- [ ] Add/Update specs
- [x] Visual design
- [x] Implementation
- [x] Write tests (unit and e2e) 
- [x] Code review
- [ ] Verify acceptance criteria met
- [x] Exploratory testing to find bugs, edge cases and verify adherence to visual design
- [ ] Write automated regression tests for the A/Cs and add to spec doc (expanding on A/Cs where possible)

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #3394 

# Description ℹ️

Update size formatting on iceberg fields and show side of order

# Demo 📺

Screenshot 2023-08-10 at 13 44 43

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

File: libs/orders/src/lib/components/order-list/order-view-dialog.tsx [hunk 1]
```diff
@@ -206,6 +222,23 @@ export const OrderViewDialog = ({
           {order.icebergOrder ? t('Yes') : t('-')}
         </div>
       </KeyValueTableRow>
+      {order.icebergOrder && (
+        <KeyValueTableRow key={'order-side'} className="ml-4">
+          <div data-testid={'order-side-label'}>{t('Side')}</div>
```
Comment (MadalinaRaicu) [inline libs/orders/src/lib/components/order-list/order-view-dialog.tsx:227]: Is it worth showing Side twice? @bwallacee

**Review by macqbat — APPROVED** [review]
